src/game.cpp

```diff
@@ -6724,8 +6725,20 @@ static void zones_manager_shortcuts( const catacurses::window &w_info, faction_i
     shortcut_print( w_info, point( tmpx, 4 ), c_white, c_light_green, _( "<Enter>-Edit" ) );
 
     tmpx = 1;
-    tmpx += shortcut_print( w_info, point( tmpx, 5 ), c_white, c_light_green,
-                            _( "<S>how all / hide distant" ) ) + 2;
+    std::string all_zones = "<S>how all";
+    std::string distant_zones = "Hide distant";
```

Comment:
```suggestion
    std::string all_zones = _( "<S>how all" );
    std::string distant_zones = _( "Hide distant" );
```
Marks the strings for translation.